chapter2/dynamic_pads.md

```diff
@@ -0,0 +1,38 @@
+# Dynamic Pads
+
+Dynamic pads, are pads that are created on demand when linking elements.
+
+## Why do we need dynamic pads?
+
+In some applications manually specifying pads isn't good enough.
+Let's consider sound mixer. If we are talking about physical piece
```

Comment:
`Piece` seems off to me. Did you mean physical device?

Reply:
https://www.merriam-webster.com/dictionary/piece
11th meaning.

Yup I meant physical device.